CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [2.0.8] - 2022-08-03
```

Comment:
Delete line 10, leave the rest under UNRELEASED. The bot will do it automatically.